pkgs/tools/backup/btrbk/default.nix

```diff
@@ -36,15 +54,24 @@ stdenv.mkDerivation rec {
   '';
 
   preFixup = ''
-    wrapProgram $out/sbin/btrbk \
+    wrapProgram $out/bin/btrbk \
       --set PERL5LIB $PERL5LIB \
+      --run 'export program_name=$0' \
       --prefix PATH ':' "${lib.makeBinPath [ btrfs-progs bash mbuffer openssh ]}"
   '';
 
+  passthru.updateScript = genericUpdater {
+    inherit pname version;
+    versionLister = writeShellScript "btrbk-versionLister" ''
+      echo "# Versions for $1:" >> "$2"
+      ${curl}/bin/curl -s https://digint.ch/btrbk/download.html | ${perl}/bin/perl -lne 'print $1 if /<li>btrbk-([0-9.]*) /'
```

Comment:
How about we use [`list-git-tags`](https://github.com/asymmetric/nixpkgs/tree/134b5c9ac026465700678036250b63702eea1ff7/pkgs/common-updater/scripts/list-git-tags) instead? Seems less fragile than parsing HTML output.

See [here](https://github.com/asymmetric/nixpkgs/blob/134b5c9ac026465700678036250b63702eea1ff7/pkgs/applications/video/shotcut/default.nix#L82) for an example.

Reply:
I'm the pleased author of the example :-)
My thought is to parse download.html because it is expected to harmonize with the src attribute and because I'm always happy to have not a github reference.

Reply:
How about https://digint.ch/download/btrbk/releases/ ?

I appreciate the intent on relying (a tiny bit) less on GH, but I think parsing HTML is prone to breakage, and it would be hard to spot.

Reply:
Like
`curl -s https://digint.ch/download/btrbk/releases/ | perl -lne 'print $1 if /btrbk-([0-9.]*)\.tar/'`
?